music_assistant/controllers/player_queues.py

```diff
@@ -2173,7 +2177,7 @@ def _handle_playback_progress_report(
             data=MediaItemPlaybackProgressReport(
                 uri=item_to_report.media_item.uri,
                 media_type=item_to_report.media_item.media_type,
-                name=item_to_report.media_item.name,
+                name=track_name,
```

Comment:
Maybe we should pass the version separate and make the scrobbler (or the helper) decide to append it to the scrobbled name?
I think that would make more sense, and also keeps it open for extension more. For example to make this a configuration option, instead of forcing it.

Reply:
I like this suggestion, let each scrobbling plugin decide what to do with the extra information.

Reply:
Will just have to make a change to the models which is easy enough.

Reply:
New fields added